Build/gulp-utils/gulp-lcl-js/index.js

```diff
@@ -0,0 +1,81 @@
+const { DOMParser } = require('xmldom');
+var through2 = require('through2');
+
+module.exports = function(opt) {
+  opt = opt || {};
+
+  // Convert LCL XML to JSON
+  var doConvert = async function(file) {
+    var xml = file.contents.toString('utf8');
+    return new Promise((resolve, reject) => {
+      try {
+        const parser = new DOMParser();
+        const doc = parser.parseFromString(xml, 'text/xml');
+        const items = doc.getElementsByTagName('Item');
+        const result = {};
+
+        for (let i = 0; i < items.length; i++) {
+          const item = items[i];
+          const itemId = item.getAttribute('ItemId');
+          
+          // Skip if no ItemId or doesn't start with semicolon
+          if (!itemId || !itemId.startsWith(';')) {
+            continue;
+          }
+
+          // Extract the key name (remove leading semicolon)
+          const key = itemId.substring(1);
+          
+          // Skip structural items that are not actual translations
+          if (key === 'Resources.resx' || key === 'Strings') {
+            continue;
+          }
+          
+          // Find the Tgt element within this item
+          const tgtElements = item.getElementsByTagName('Tgt');
+          if (tgtElements.length > 0) {
+            const tgtElement = tgtElements[0];
+            const valElements = tgtElement.getElementsByTagName('Val');
```

Comment:
[Checking] If XML has unexpected structure (e.g., missing <Val> but has <Tgt>) the code silently skips, is that intended?